native-rating/rate-card-manager.md

```diff
@@ -0,0 +1,535 @@
+---
+title: Rate Card Manager
+---
+
+# Rate Card Manager
+
+The Rate Card Manager (RCM) is a system that allows customers to bring their own rate cards for use in ShipStation and ShipEngine. For a carrier to support this, it needs to use [Native Rating](./index.md) and have its rating logic written so that it can handle the pre-defined data and key formats generated by the RCM. After familiarizing yourself with building a [Native Rating implementation](./implementation.md) and building the [rating logic](./rating-logic.md), continue on to learn about the details required to support the RCM.
```

Comment:
not sure this supports shipstation.. 🤔

Reply:
I'd assumed this all supported both because I hadn't seen anything that explicitly excluded ShipStation from working with RCM carriers.  I can take out references to ShipStation if that's the case.

Reply:
I haven't seen anything either, but don't know if they can do calls to the API without the shipstation UI though.

Reply:
We could say Shipengine API users (or something like that), I think this would include Shipstation or any other solution of the company that wants to use Shipengine API.